hack/jenkins/job-configs/kubernetes-e2e.yaml

```diff
@@ -1,5 +1,17 @@
-- job-template:
-    name: 'kubernetes-e2e-{suffix}'
+# Common publishers that all Jobs share.
+- publisher:
+    name: common-publishers
```

Comment:
Might call this `e2e-publishers`, since not *all* jobs use these. When you do `jenkins-jobs update hack/jenkins/job-configs` there's not really any file-based namespaces.